### PR TITLE
ADCM-2731 add hostcomponentmap in cluster view

### PR DESCRIPTION
**Pull request by SofiaTal**

### CODE REVIEW

**Comment by SofiaTal** [comment]
bender build

**Comment by BenderArenadata** [comment]
Hey, @SofiaTal! Build has been launched!

**Comment by SofiaTal** [comment]
bender build

**Comment by BenderArenadata** [comment]
Hey, @SofiaTal! Build has been launched!

File: python/api/serializers.py [hunk 1]
```diff
@@ -69,14 +69,29 @@ def get_config(self, instance):
...
             return {'cluster_id': self.context['cluster_id'], 'upgrade_id': obj.id}
 
+    hostcomponentmap = serializers.SerializerMethodField()
     url = MyUrlField(read_only=True, view_name='cluster-upgrade-details')
     do = MyUrlField(read_only=True, view_name='do-cluster-upgrade')
 
+    def get_hostcomponentmap(self, instance):
+        if instance.action:
+            return instance.action.hostcomponentmap
+        return {}
```
Comment (a-alferov) [inline python/api/serializers.py:84]: This should be a list `[]`

**Review by a-alferov — CHANGES_REQUESTED** [review]

File: python/api/serializers.py (hunk 1 above)
Comment (SofiaTal) [reply python/api/serializers.py]: Done in last commit

**Review by a-alferov — APPROVED** [review]

**Pull request merged**